samples/spring-eventsourced-customer-registry/src/main/java/customer/view/CustomerView.java

```diff
@@ -0,0 +1,59 @@
+package customer.view;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import customer.api.*;
+
+import java.util.Optional;
+
+public class CustomerView {
+  final public String email;
+  final public String name;
+  final public Address address;
+
+  @JsonCreator
+  public CustomerView(@JsonProperty("email") String email, @JsonProperty("name") String name, @JsonProperty("address") Address address) {
```

Comment:
```suggestion
  public CustomerView(String email, String name, Address address) {
```
going all-in at this point...